tc/sgx/trusted_worker_manager/enclave/work_order_processor.cpp

```diff
@@ -349,12 +375,13 @@ namespace tcf {
         }
         // Calculate final hash
         std::string final_hash_string = ByteArrayToString(hash_1);
-        if(!hash_in_data_str.empty()) {
+
+        //if(!hash_in_data_str.empty()) {
```

Comment:
Is there a reason this is commented out and not deleted?

Reply:
Its irrelevant change. Deleted it